tools/review.py

```diff
@@ -21,7 +21,7 @@ def Main():
   argument_parser = argparse.ArgumentParser(
       description='Script to manage code reviews.')
 
-  # TODO: add option to directly pass code review issue number.
+  # TODO: add option to directly pass pull request issue number.
```

Comment:
Is this todo still needed? I don't think it is. Pull requests always operate on the branch, so there's no need to differentiate between the pull request issue number and the branch name.